template/chat.html

```diff
@@ -34,6 +34,9 @@
         }
       },
 
+      /*
+        writes a message in the chat box.
+      */
       sendMessage: function(user, message) {
         if (message != '')
```

Comment:
This is q dangerous check to do. If message == "  " (two whitespaces), the test will pass. So, as it is, it implies at least that the string has been trimmed beforehand. 
Otherwise, we should use underscore.js, underscore.string.js plug-in (https://github.com/epeli/underscore.string) and the isBlank method for strings (that avoids that we have to rewrite our own).